src/app/component/layout/components/search-popup/search-popup.component.html

```diff
@@ -54,7 +54,7 @@ <h3>{{ "search.search-popup.tips" | translate }}</h3>
           </a>
         </ng-template>
 
-        <ng-template [ngIf]="(isNewsSearchFound === false && isTipsSearchFound === false)">
+        <ng-template [ngIf]="(!isNewsSearchFound && !isTipsSearchFound)">
           <br/><br/>
```

Comment:
`br` should be remove - it's bad practice to use the `br` to format your document or to add space between elements

Reply:
fixed